e2e/basic/pages/index.md

```diff
@@ -34,7 +34,7 @@ This page can be found in your project at `/pages/index.md`. Make a change to th
       category
   from needful_things.orders
   where category like '${inputs.category.value}'
-  and date_part('year', order_datetime) like '${inputs.year.value}'
+  and cast(date_part('year', order_datetime) as varchar) like '${inputs.year.value}'
```

Comment:
what's going on here?
are users going to have to update their code if they use date_part or date_trunc?

Reply:
Yeah `date_part` returns a number, and doesn't seem like duckdb allows for the implicit conversion from number to string for the comparison with `'${inputs.year.value}'` anymore

Reply:
Actually just realized this exists: https://duckdb.org/docs/configuration/pragmas.html#implicit-casting-to-varchar

Reply:
don't think there's any user-facing breaking changes after that fix